### ISSUE TITLE
Allow return in multiple-line methods

### ISSUE THREAD

**Issue by khiav223577** [bbatsov/rubocop#4504]
The `RedundantReturn` cop doesn't allow you to specify a return statement in a method more than one-line. I think redundant return is acceptable (and looks prettier) in a method like this:
```rb
def foo
  return aaa if condition1
  return bbb if condition2
  return ccc if condition3
  return ddd if condition4
end
```

**Comment by jonas054** [comment]
The given example is not reported as an offense as far as I can see. I'm closing the issue as invalid. Let me know if I missed something.
